bowtie/_report.py

```diff
@@ -52,6 +54,32 @@ class MissingFooter(InvalidReport):
     """
 
 
+class CaseMismatchInSameSuiteReports(Exception):
```

Comment:
I'll have to read where you use these, but I find the docstrings a bit hard to understand (i.e. to understand what you mean to say will cause these).